.gitignore

```diff
@@ -37,6 +37,6 @@ bower_components
 *.pem
 
 # Temporal CLI test data
-tests/cli/temporal
+temporal-tests
```

Comment:
I moved the temporal tests location to make it easier to exclude them from linting